### DISCUSSION TITLE
Move lists should be restricted to only the characters on screen.

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#2115] [Feature Request]
As it is, during team matches Ikemen allows you to browse the move list of every character. I think it would be more logical to only display the move lists of the characters currently on screen.

**Comment by potsmugen** [comment]
During tag it makes sense to be able to access the partner movelists without needing to tag them in. But I'm not sure. This should be researched in modern games.

**Reply by UrsoManso** [reply]
The only team mode that this behavior actually makes sense to me is simul tbh.

**Comment by formula1** [comment]
Maybe it's better off being an option to turn on or off
